README.md

```diff
@@ -40,16 +40,40 @@ Configuration
 
 To use this extension, simply add the following code in your application configuration:
 
+Using IBM DB2:
+
 ```php
 return [
     //....
     'components' => [
         'db' => [
-            'class'    => 'edgardmessias\db\ibm\db2\Connection',
-            'dsn'      => 'ibm:DRIVER={IBM DB2 ODBC DRIVER};DATABASE=test;"HOSTNAME=127.0.0.1;PORT=50000;PROTOCOL=TCPIP',
-            'username' => 'username',
-            'password' => 'password',
+            'class'         => 'edgardmessias\db\ibm\db2\Connection',
+            'dsn'           => 'ibm:DRIVER={IBM DB2 ODBC DRIVER};DATABASE=test;"HOSTNAME=127.0.0.1;PORT=50000;PROTOCOL=TCPIP',
+            'username'      => 'username',
+            'password'      => 'password',
+            'defaultSchema' => '',
+            'isISeries'     => false
         ],
     ],
 ];
 ```
+
+Using ODBC IBM iAccess driver:
+
+```php
+return [
+    //....
+    'components' => [
+        'db' => [
+            'class'         => 'edgardmessias\db\ibm\db2\Connection',
+            'dsn'           => 'odbc:DRIVER={IBM i Access ODBC Driver 64-bit};SYSTEM=127.0.0.1;PROTOCOL=TCPIP',
+            'username'      => 'username',
+            'password'      => 'password',
+            'defaultSchema' => '',
+            'isISeries'     => false
```

Comment:
Schuld be true only if on iSeries system

Cordiali saluti
-- 
Marco Migozzi

Da: Edgard Lorraine Messias <notifications@github.com>
Rispondi: edgardmessias/yii2-ibm-db2 <reply@reply.github.com>
Data: 1 marzo 2016 at 12:03:50
A: edgardmessias/yii2-ibm-db2 <yii2-ibm-db2@noreply.github.com>
Cc: ToRvaLDz <torvaldz@gmail.com>
Oggetto:  Re: [yii2-ibm-db2] Added support for iSeries (#1)  

In README.md:

>          ],
>      ],
>  ];
>  ```
> +
> +Using ODBC IBM iAccess driver:
> +
> +```php
> +return [
> +    //....
> +    'components' => [
> +        'db' => [
> +            'class'         => 'edgardmessias\db\ibm\db2\Connection',
> +            'dsn'           => 'odbc:DRIVER={IBM i Access ODBC Driver 64-bit};SYSTEM=127.0.0.1;PROTOCOL=TCPIP',
> +            'username'      => 'username',
> +            'password'      => 'password',
> +            'defaultSchema' => '',
> +            'isISeries'     => false
Should not it be true?

—
Reply to this email directly or view it on GitHub.